sist02.csl

```diff
@@ -8,199 +8,133 @@
     <author>
       <name>Shoji Takahashi</name>
       <email>s.takahashi@elsevier.com</email>
-      <uri>http://www.mendeley.com/profiles/shoji-takahashi/</uri>
+      <uri>http://www.mendeley.com/profiles/shoji-takahashi3/</uri>
     </author>
     <contributor>
       <name>Eriko Amano</name>
       <uri>http://www.mendeley.com/profiles/eriko-amano/</uri>
     </contributor>
     <category citation-format="numeric"/>
-    <updated>2014-11-26T05:09:41+00:00</updated>
+    <updated>2015-12-13T06:04:50+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
   <locale xml:lang="ja">
     <terms>
-      <term name="chapter" form="short">ch.</term>
-      <term name="presented at">presented at the</term>
-      <term name="available at">available</term>
+      <term name="open-quote">“</term>
+      <term name="close-quote">”</term>
     </terms>
   </locale>
   <macro name="author">
     <names variable="author" suffix=". ">
       <name initialize="false" initialize-with=". " name-as-sort-order="all"/>
-      <label form="short" prefix=", " text-case="capitalize-first"/>
-      <substitute>
-        <names variable="translator"/>
-      </substitute>
     </names>
   </macro>
-  <macro name="edition">
-    <number prefix="第" variable="edition"/>
-    <text term="edition" form="short" suffix=", "/>
-  </macro>
   <macro name="event">
-    <text variable="event-place" suffix=", "/>
-    <date delimiter="" variable="event-date" suffix=". ">
-      <date-part name="year" range-delimiter="" suffix="-"/>
-      <date-part name="month" form="numeric-leading-zeros" range-delimiter="" suffix="-"/>
-      <date-part name="day" form="numeric-leading-zeros" range-delimiter=""/>
-    </date>
+    <group delimiter=", ">
+      <text variable="event-place"/>
+      <date variable="event-date">
+        <date-part name="year" suffix="-"/>
+        <date-part name="month" form="numeric-leading-zeros" suffix="-"/>
+        <date-part name="day" form="numeric-leading-zeros"/>
+      </date>
+    </group>
   </macro>
   <macro name="title">
     <choose>
-      <if type="article-journal" match="any">
-        <text variable="collection-title" suffix=": "/>
-        <text variable="title" suffix=". "/>
-        <text variable="container-title"/>
-      </if>
-      <else-if type="chapter paper-conference report" match="any">
+      <if type="chapter report paper-conference article webpage" match="any">
         <choose>
-          <if match="any" variable="container-title">
-            <text variable="title" prefix="&quot;" suffix="&quot;. "/>
-            <text variable="container-title" suffix=". "/>
+          <if match="any" variable="container-title" type="webpage">
+            <text term="open-quote"/>
+            <text variable="title" quotes="false"/>
```

Comment:
Hi,

Thanks.  I thought you would catch this.  Let me explain.

What I am trying to get here is a title enclosed with quotes followed by a comma like below.
“TITLE”,

FYI, locales-ja-JP.xml has <style-options punctuation-in-quote="false"/>.

1.
With the current coding, I can get the desired result in CSL Editor, Mendeley Desktop, and Word (see the attached sist02-1.png).
I added the <locale xml:lang="ja"> section because locales-ja-JP.xml has Japanese parentheses 『 』.
<locale xml:lang="ja">
  <terms>
    <term name="open-quote">“</term>
    <term name="close-quote">”</term>
  </terms>
</locale>
…
<choose>
   <if match="any" variable="container-title" type="webpage">
      <text term="open-quote"/>
      <text variable="title" quotes="false"/>
      <text term="close-quote"/>
    </if>
    <else>
      <text variable="title"/>
      </else>
</choose>

2.
When you use quotes="true", you get the desired result in CSL Editor but not in Mendeley Desktop or Word (see attached sist02-2.png).
<locale xml:lang="ja">
  <terms>
    <term name="open-quote">“</term>
    <term name="close-quote">”</term>
  </terms>
</locale>
…
<choose>
  <if match="any" variable="container-title" type="webpage">
     <text variable="title" quotes="true"/>
  </if>
  <else>
     <text variable="title"/>
  </else>
</choose>

3.
I also tried the following, without the <locale xml:lang="ja"> section this time, but again I get the desired result in CSL Editor but not in Mendeley Desktop or Word (see attached sist02-3.png).
<choose>
  <if match="any" variable="container-title" type="webpage">
    <text variable="title" quotes="false" prefix="&quot;" suffix="&quot;"/>
  </if>
  <else>
     <text variable="title"/>
  </else>
</choose>

Hope this explains the background of my coding.

Best regards,

Shoji

Shoji Takahashi
Solution Consultant (e-Platform and Content), A&G
Elsevier Japan KK
Tel: +81-3-5561-5994
Fax: +81-3-5561-0451
Address:
Higashi-Azabu 1-Chome Bldg. 4F
1-9-15 Higashi-Azabu, Minato-ku, Tokyo
106-0044 JAPAN




From: Sebastian Karcher [mailto:notifications@github.com]
Sent: Monday, December 14, 2015 1:18 AM
To: citation-style-language/styles
Cc: Takahashi, Shoji (ELS-TOK)
Subject: Re: [styles] Update sist02.csl (#1828)


In sist02.csl<https://github.com/citation-style-language/styles/pull/1828#discussion_r47446658>:

>          <choose>

> -          <if match="any" variable="container-title">

> -            <text variable="title" prefix="&quot;" suffix="&quot;. "/>

> -            <text variable="container-title" suffix=". "/>

> +          <if match="any" variable="container-title" type="webpage">

> +            <text term="open-quote"/>

> +            <text variable="title" quotes="false"/>

why aren't you just doing quotes="true" here instead?

—
Reply to this email directly or view it on GitHub<https://github.com/citation-style-language/styles/pull/1828/files#r47446658>.